04_thermal_conductivity/README.md

```diff
@@ -0,0 +1,456 @@
+plt.savefig('Spectral_thermal_conductivity_MgO.png')
+plt.show()
+#plot the cumulative thermal conductivity as a function of the total mean free path
+plt.semilogx(fn['temperature_1']['mean_free_path_axis'][:], 
+             fn['temperature_1']['cumulative_kappa_vs_mean_free_path_total'][:], lw = 4)
+#add labels, titles etc.
+plt.title('Cumulative kappa vs mean free path of MgO')
+plt.xlabel('Mean Free Path [m]')
+plt.ylabel(r'$\kappa$ [W/K/m]')
+plt.xlim(1E-9,1E-5)
+plt.savefig('thermal_conductivity_vs_mfp_MgO.png')
+```
+The script should give you the plots reported below. 
+![Spectral thermal conductivity of MgO](https://github.com/RobertaFarris93/tdep-tutorials/tree/thermal_conductivity/04_thermal_conductivity/Plots/Spectral_thermal_conductivity_MgO.png)
+
+
+![Cumulative kappa vs mean free path of MgO](https://github.com/RobertaFarris93/tdep-tutorials/tree/thermal_conductivity/04_thermal_conductivity/Plots/thermal_conductivity_vs_mfp_MgO.png)
+
+
+# Convergence of thermal conductivity 
+
+So far we consider converged the forces used for the thermal conductivity. As for the other physical quantities, the thermal conductivity needs to be tested against all the parameters used in the calculations. In particular:
+
+* supercell size
+* number of configurations used
+* range of the forces cutoffs (number of neighbours used included in the integral)
+* number of iterations in the self-consistent loop 
+
+## Supercell size convergence
+
+you can test the size of the supercells used by following the steps explained in Tutorial 01 for the case of MgO.
+
+
+## Self-consistent loop
+
+In order to converge the thermal conductivity, we should test it against the sampling in a iterative way. For doing that, we should repeat the steps explained in the Tutorial 01 and test the goodness of our fit for the desired property, in this case the thermal conductivity (sTDEP scheme). 
+
+To do so, with the data provided in the folder ``convergence_tests/input_MgO/``` we can:
+
+
+* Create a set of canonical configurations using:
+  ```
+  canonical_configuration --quantum  --temperature 300 -n 3
+   ```
+  Here, we are using an initial set of IFCs in order to create a set of configurations, in case you want to start from scratch without an initial set of forceconstants, you can use two flags ```--debye_temperature ``` and  ``` --maximum_frequency```. For the details, read the documentation on the [canonical_configuration](https://ollehellman.github.io/program/canonical_configuration.html) .
+  
+  You should see now three configurations in your folder: `
+  ``contcar_conf0001```
+  
+   ```contcar_conf0002```
+  
+  ```contcar_conf0003```
+
+* Compute the atomic forces using a DFT code of your choice.
+* 
+  **Tip**: in order to avoid this step, that could require a significant amount of time, we provided a potential for MgO. You can download that from the first Tutorial. Create a folder "iter0" and copy your input files and the potential there.
+
+  Then run:
```

Comment:
we should link to the dedicated tutorials

Reply:
suggestion:
- perform self-consistent loop as described in https://github.com/tdep-developers/tdep-tutorials/tree/main/01_sampling for MgO (**I will add that tutorial explicitly**) for 10 iterations
- compute thermal conductivity at each step
- study convergence